how-to.v2/use-a-sidecar-app-basic/platform/public/js/provider.bundle.js

```diff
@@ -0,0 +1,228 @@
+/******/ (() => { // webpackBootstrap
```

Comment:
Should this file be excluded in the gitignore

Reply:
added to gitignore